External/pubchem_shape/test.cpp

```diff
@@ -282,3 +282,22 @@ TEST_CASE("Github #8096") {
     CHECK_THAT(nbr_ct, Catch::Matchers::WithinAbs(1.0, 0.005));
   }
 }
+
+TEST_CASE("Serialization") {
```

Comment:
This needs to also be inside a test for whether or not serialization is enabled.